src/main/kotlin/ink/abb/pogo/scraper/gui/SocketServer.kt

```diff
@@ -63,7 +63,14 @@ class SocketServer {
             profile.levelRatio = ratio
             profile.pokebank = ctx!!.api.inventories.pokebank.pokemons.size
             profile.pokebankMax = ctx!!.api.playerProfile.pokemonStorage
-            profile.items = ctx!!.api.inventories.itemBag.items.size
+
+            // Replace with @pr0ves method
+            var count = 0
+            ctx!!.api.inventories.itemBag.items.toMutableList().forEach {
+                count += it.count
+            }
+            profile.items = count
+
```

Comment:
You can just use `ctx!!.api.inventories.itemBag.size()` directly, without copypasting my function here.